cmd/context/use.go

```diff
@@ -61,6 +63,11 @@ Or a Kubernetes context:
 				ctxOptions.Context = strings.TrimSuffix(args[0], "/")
 			}
 
+			switch ctxOptions.Origin {
+			case login.CLIOrigin, login.DockerDesktopOrigin:
+			default:
+				return fmt.Errorf("'origin' mus be either '%s' or '%s'", login.CLIOrigin, login.DockerDesktopOrigin)
```

Comment:
the VSCode extension sets a different origin

Reply:
the docker desktop extension doesn't support env vars at command execution, that's why we need to use a `hidden` flag for now